internal/k8shandler/kibana/reconciler.go

```diff
@@ -89,8 +90,22 @@ func Reconcile(requestCluster *kibana.Kibana, requestClient client.Client, esCli
 		return err
 	}
 
+	clusterUUID := uuid.New().String()
```

Comment:
not sure how strict uuid is to Kibana, but a more predictable pattern so we do not have to stash it someplace:
* md5 the name or ns/name
* if there is a way to predictably generate a uuid from a seed value